CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/GiniColors.swift

```diff
@@ -57,7 +57,6 @@ extension UIColor {
         static let helpBackground = UIColorPreferred(named: "helpBackground")
         static let separator = UIColorPreferred(named: "separator")
         static let subheadline = UIColorPreferred(named: "subheadline")
-        static let systemBlue = UIColorPreferred(named: "systemBlue")
```

Comment:
If you remove a color from here, make sure to remove the color asset as well

Reply:
I was thinking we will do the cleanup for all colors? @chriskryn could you please put it back 😉